<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasApplianceElements</class>
 <widget class="QWidget" name="CanvasApplianceElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>314</height>
   </rect>
  </property>
  <widget class="GroupBoxWidget" name="addContentBox" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>252</width>
     <height>50</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title" stdset="0">
    <string>INSERT</string>
   </property>
   <property name="borderFlags" stdset="0">
    <number>2</number>
   </property>
   <layout class="QGridLayout" name="addContentLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="PixmapButton" name="bPicture" native="true">
      <property name="toolTip">
       <string>Add Picture</string>
      </property>
      <property name="fixedSize" stdset="0">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="pixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-image.png</pixmap>
      </property>
      <property name="hoverPixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-overlay-plus.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="PixmapButton" name="bText" native="true">
      <property name="toolTip">
       <string>Add Text</string>
      </property>
      <property name="fixedSize" stdset="0">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="pixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-text.png</pixmap>
      </property>
      <property name="hoverPixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-overlay-plus.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="addWebcamLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="PixmapButton" name="bWordcloud" native="true">
      <property name="toolTip">
       <string>Add Wordcloud</string>
      </property>
      <property name="fixedSize" stdset="0">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="pixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-wordcloud.png</pixmap>
      </property>
      <property name="hoverPixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-overlay-plus.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="PixmapButton" name="bCanvas" native="true">
      <property name="toolTip">
       <string>Add Canvas</string>
      </property>
      <property name="fixedSize" stdset="0">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="pixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-fotowall.png</pixmap>
      </property>
      <property name="hoverPixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-overlay-plus.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="PixmapButton" name="bWebsearch" native="true">
      <property name="toolTip">
       <string>Search Web Pictures</string>
      </property>
      <property name="fixedSize" stdset="0">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="pixmap" stdset="0">
       <pixmap resource="../fotowall.qrc">:/data/insert-download.png</pixmap>
      </property>
      <property name="checkable" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>157</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">This is the 'add content' Box</string>
   </property>
  </widget>
  <widget class="GroupBoxWidget" name="propertiesBox" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>12</width>
     <height>50</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="borderFlags" stdset="0">
    <number>0</number>
   </property>
   <layout class="QHBoxLayout" name="propLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>197</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">This is the 'Content Properties' Box</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>127</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">This is the Canvas Box</string>
   </property>
  </widget>
  <widget class="GroupBoxWidget" name="canvasPropertiesBox" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>189</width>
     <height>44</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="16">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title" stdset="0">
    <string>CANVAS</string>
   </property>
   <property name="checkable" stdset="0">
    <bool>false</bool>
   </property>
   <property name="borderFlags" stdset="0">
    <number>0</number>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QToolButton" name="backButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Background</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="arrangeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Arrange</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="undoButton">
      <property name="toolTip">
       <string>Undo</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="undo">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Z</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="decoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Decorations</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="projectCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Screen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Exact Size...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desktop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CD Cover</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DVD Cover</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="redoButton">
      <property name="toolTip">
       <string>Redo</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="redo">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+Z</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="GroupBoxWidget" name="fileBox" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>56</width>
     <height>47</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>238</green>
         <blue>216</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="32">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>238</green>
         <blue>216</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="32">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="32">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="32">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title" stdset="0">
    <string>FILE</string>
   </property>
   <property name="checkable" stdset="0">
    <bool>false</bool>
   </property>
   <property name="borderFlags" stdset="0">
    <number>1</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="exportButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../fotowall.qrc">
        <normaloff>:/data/fotowall.png</normaloff>:/data/fotowall.png</iconset>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>103</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">This is the File Box</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxWidget</class>
   <extends>QWidget</extends>
   <header>Shared/GroupBoxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PixmapButton</class>
   <extends>QWidget</extends>
   <header>Shared/PixmapButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../fotowall.qrc"/>
 </resources>
 <connections/>
</ui>
